docs/02-guides/display-wysiwyg-content.mdx

```diff
@@ -0,0 +1,344 @@
+large bundles on pages that don't need them.
+
+:::
+
+### Fetching data to render Wysiwyg components
+
+You can now transform HTML tags into React components. However, in some cases
+the data provided in the raw HTML you've received from your remote service won't
+be enough to display your component. For instance, you may have a
+`<product-name sku="VSK12" />` that is supposed to display the product's name.
+But all you have in the attributes is the SKU. This can be done by associating a
+`WysiwygNodeData` with your HTML node.
+
+Indeed, when you've created your GraphQL `CustomWysiwyg` type, you have
+declared: `childNodes` (`JSON`) and `data` (`[WysiwygNodeData]`). By default,
+`data` will always be empty. But you can tell the Wysiwyg loader to fetch data
+for some specific `childNodes` (that each represent an HTML tag). This means
+that for `product-name`, you will associate a new `data` element that will
+contain the whole product data needed to display the component. To do so, you
+need to follow these steps:
+
+#### Add the product data to your GraphQL schema
+
+1.  Make sure that your dependencies are up to date in your GraphQL module
+    ```ts title="extensions/acme-extension/modules/wysiwyg/index.ts"
+    export default createGraphQLModule({
+      namespace: "Magento2/Cms",
+      // highlight-start
+      dependencies: [
+        "Front-Commerce/Wysiwyg", // ensure that Wysiwyg related features are available
+        "Magento2/Catalog/Product" // ensure that you can fetch a product in your Wysiwyg data
+      ],
+      // highlight-end
+    ```
+1.  Define a new `WysiwygProductNameData` type that implements `WysiwygNodeData`
+    and expose all the data you need to display your component
+    ```ts title="extensions/acme-extension/modules/wysiwyg/index.ts"
+    typeDefs: /* GraphQL */ `
+      type WysiwygProductNameData implements WysiwygNodeData {
+        dataId: ID
+        product: Product
+      }
+    `,
+    ```
+1.  Setup the resolvers to tell GraphQL how to fetch the `product` field in your
+    new `WysiwygProductNameData` type
+    ```ts title="extensions/acme-extension/modules/wysiwyg/resolvers.js"
+    export default {
+      WysiwygProductNameData: {
+        product: ({ node }, _, { loaders }) => {
+          const productSkuAttribute = node.attrs.find(
+            ({ name }) => name === "sku"
+          );
+          if (!productSkuAttribute) {
+            return null;
+          }
+          return loaders.Product.load(productSkuAttribute.value);
+        },
+      },
+    };
+    ```
+1.  Register the HTML tag and associate it with the GraphQL type
+    ```ts title="extensions/acme-extension/modules/wysiwyg/runtime.ts"
+    // ...
+    contextEnhancer: ({ loaders }) => {
+      loaders.Wysiwyg.registerNodeType(
+        "product-name", // the tag name in your HTML
+        "WysiwygProductNameData" // The associated GraphQL type name
+      );
+    };
+    // ...
+    ```
```

Comment:
As previously mentioned, inline the resolvers in the runtime, and wrap with the creator function